unfoldingWord/translationCore
# Feature mcleanb 3086/show invalidated selection.4

#### Describe what your pull request addresses (Please include relevant issue numbers):
- updates UI toolkit version to 0.9.2

#### Please include detailed Test instructions for your pull request:
- uses https://github.com/translationCoreApps/translationWords/pull/125 
- uses https://github.com/translationCoreApps/tc-ui-toolkit/pull/15

- Local testing setup: 
  `git fetch`
  `git checkout feature-mcleanb-3086/ShowInvalidatedSelection.4`
  `npm i`
  `cd tC_apps/translationWords`
  `git fetch`
  `git checkout feature-mcleanb-3086/ShowInvalidatedSelection.2`
  `cd ../..`
  `npm start`

- Test steps:
  - open tw tool.  Under blameless, for vs 1:6 selected "without blame" then clicked on vs 1:7, using scripture pane editor edited 1:6 to remove space to make selection "withoutblame".  Clicked on 1:6. see invalidated icon on group menu and on verse check bar and on instructions.

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/4604)
